### DISCUSSION TITLE
argo-workflows/intermediate-inputs/

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#10500] [General]
# argo-workflows/intermediate-inputs/

https://argoproj.github.io/argo-workflows/intermediate-inputs/

**Comment by caseyjmorton** [comment]
Is it possible for a user to select multiple values?

**Comment by caseyjmorton** [comment]
also, is it possible to mask inputs for things like passwords, etc?

**Reply by agilgur5** [reply]
Not currently. #11431 is an open enhancement request for this.

Mounting Secrets is a better way around this if you're able to do so.

**Comment by schlunsen** [comment]
Is it possible to pass the input arguments into the job as environment variables?

**Reply by agilgur5** [reply]
Yes you should be able to do so. For more details, see https://argoproj.github.io/argo-workflows/variables/

**Comment by lazure-ocean** [comment]
How to use expressions in inputs?
